packages/cypress/src/executors/cypress/cypress.impl.ts

```diff
@@ -144,6 +147,20 @@ A generator to migrate from v8 to v10 is provided. See https://nx.dev/cypress/v1
   }
 }
 
+function assertCypressInstalled() {
```

Comment:
Do we need this? I thought `cypress` does the same thing.

Reply:
I can remove it.

there seems to be issue where cypress doesn't install in CI with npm at times (as is reproduce-able by our nightly tests). but @meeroslav mentioned maybe we just do it in our e2e suite workspace creation. could just be something in only our env causing npm to not run the postinstall hook from cypress package. since it hasn't been reported by the community (linked issue is just the init generator wasn't being ran so the dep was never installed, vs installed but binary not downloaded)

Reply:
went ahead and removed and we can add it back if there are reports of cypress not being installed outside of our e2e tests.